docs/sce_file_contents.md

```diff
@@ -0,0 +1,131 @@
+
+### Experiment metadata
+
+Metadata associated with data processing is included in the `metadata` slot as a list.
+
+```r
+expt_metadata <- metadata(sce)
+```
+
+| Item name           | Contents                                                                                                 |
+| ------------------- | -------------------------------------------------------------------------------------------------------- |
+| `salmon_version`    | Version of `salmon` used for initial mapping                                                             |
+| `reference_index`   | Transcriptome reference file used for mapping                                                            |
+| `total_reads`       | Total number of reads processed by `salmon`                                                              |
+| `mapped_reads`      | Number of reads successfully mapped                                                                      |
+| `mapping_tool`      | Pipeline used for mapping and quantification (`alevin-fry` for all current data in ScPCA)                |
+| `alevinfry_version` | Version of `alevin-fry` used for mapping and quantification                                              |
+| `af_permit_type`    | `alevin-fry generate-permit-list` method used for filtering cell barcodes                                |
+| `af_resolution`     | `alevin-fry quant` resolution mode used                                                                  |
+| `usa_mode`          | Boolean indicating whether quantification was done using `alevin-fry` USA mode                           |
+| `af_num_cells`      | Number of cells reported by `alevin-fry`                                                                 |
+| `transcript_type`   | Transcripts included in gene counts: `spliced` for single-cell samples and `unspliced` for single-nuclei |
+
+## Additional fields for filtered cells
+
+After {ref}`filtering for empty droplets <processing_information:Filtering cells>`, some additional fields are added to the `SingleCellExperiment` object.
+
+Within `colData`, we add the column `prob_compromised`, which denotes the probability that a cell is compromised (i.e., dead or damaged), as calculated by [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html). 
+We also add to the `metadata` a `miQC_model` item, which includes the model that `miQC` fit to the data and which was used to calculate `prob_compromised`. 
+
+## Additional fields for CITE-seq samples
+
+CITE-seq data, when present, is included within the `SingleCellExperiment` as an "Alternative Experiment" named `"CITEseq"` , which can be accessed with the following command:
+
+```r
+altExp(sce, "CITEseq")
+```
+
+Within this, the main data matrix is again found in the `counts` assay, with each column corresponding to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponding to an antibody derived tag (ADT).
```

Comment:
```suggestion
Within this, the main expression matrix is again found in the `counts` assay, with each column corresponding to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponding to an antibody derived tag (ADT).

To access the counts matrix, specifically for CITE-seq data, you can use the following command: 

```r
cite_counts <- counts(altExp(sce, "CITEseq"))
```
I think it might be helpful to show here how you would do the same things in parallel to what we showed with the RNA-seq data. I know its repetitive but it could be helpful to people who don't have as much experience with working with multi-modal data in particular.

Reply:
Agree with the idea here, but I don't think of this as expression? I guess protein expression? I'll probably use expression unless I can come up with something better.